package/yast2-bootloader.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Jan 25 11:18:12 UTC 2024 - Knut Anderssen <kanderssen@suse.com>
+
+- Add cio_ignore to allowed kernel options on s390 (bsc#1210525)
```

Comment:
I would not say `allowed` and more likely mention that it will be kept from installation environment.

Reply:
Fine with me just reused the previous entry for rd.zdev.